kbase-extension/static/kbase/js/widgets/appWidgets2/input/floatInput.js

```diff
@@ -183,7 +183,7 @@ define([
                                 style: {
                                     textAlign: 'right',
                                 },
-                                value: initialControlValue,
+                                value: currentValue,
```

Comment:
Just put the string in place, even if it shouldn't be there, so the user can see what they did wrong.